update.elv

```diff
@@ -47,9 +47,9 @@ fn current-commit-or-tag {
 fn last-modified {
   platform = (uname)
   if (eq $platform "Darwin") {
-    put (/bin/date -u -j -r (/usr/bin/stat -f%B (which elvish)) +"%a, %d %b %Y %H:%M:%S GMT")
+    put (/usr/bin/env date -u -j -r (/usr/bin/env stat -f%B (which elvish)) +"%a, %d %b %Y %H:%M:%S GMT")
```

Comment:
I'm not opposed to this since it shouldn't hurt anything, but shouldn't `/bin/date` and /usr/bin/stat` both be pretty reliable to use on macOS?

Reply:
Not a MacOS user, I guess they should be but I updated all four command for the sake of consistency

Reply:
🤷‍♂️

Good enough reason for me.